src/commcare_cloud/ansible/roles/common_installs/tasks/main.yml

```diff
@@ -112,9 +106,25 @@
   notify: restart chrony
   when: use_chrony
 
-- include: bionic_apt_installs.yml
+- name: Install common packages (Bionic)
+  become: yes
+  apt:
+    name:
+      - python-minimal
+      - python3-pip
+      - software-properties-common
+      - ufw
   when: ansible_distribution_version == '18.04'
 
+- name: Install common packages (Jammy)
+  become: yes
+  apt:
+    name:
+      - acl
+      - software-properties-common
+      - ufw
```

Comment:
Can we remove already ufw at least on 22.04?

Reply:
To me that's a separate question from the Ubuntu upgrade. It's also a bigger change than I'd want to sneak into a PR like this—to the extent that any third party relies at all on this configuration to manage their port accessibility (which some almost certainly do) this is a change we would need to have very clear communication about ahead of time.